foundations/src/telemetry/log/init.rs

```diff
@@ -88,14 +89,14 @@ pub(crate) fn init(service_info: &ServiceInfo, settings: &LoggingSettings) -> Bo
             let drain = build_json_log_drain(stdout_writer);
             AsyncDrain::new(drain).chan_size(CHANNEL_SIZE).build()
         }
-        (LogOutput::File(file), LogFormat::Text) => {
-            let drain = TextDrain::new(PlainDecorator::new(File::create(file)?))
-                .build()
-                .fuse();
+        (LogOutput::File(file_path), LogFormat::Text) => {
```

Comment:
I think the original issue was observed when we were writing to stdout (i.e. `Terminal`) and not to a file. iirc we really use file outputs only for testing/debugging purposes on dev machines